app/views/stats/index.html.erb

```diff
@@ -7,10 +7,15 @@
     }
   ]
 } %>
-<h2>Deployments per month</h2>
+
+<%= render "govuk_publishing_components/components/title", {
+  title: "Deployments per month"
+} %>
 
 <%= line_chart @stats.per_month, width: "100%", height: "450px", legend: false, download: true %>
 
-<h2>Deployments per year</h2>
+<%= render "govuk_publishing_components/components/title", {
```

Comment:
Spacing here is a bit tight...

![Screenshot 2019-12-02 at 10 40 54](https://user-images.githubusercontent.com/861310/69952989-42952a80-14f0-11ea-9927-3a2dee185cf2.png)